src/libraries/System.Collections.Immutable/tests/ImmutableArrayTest.cs

```diff
@@ -838,7 +838,7 @@ public static IEnumerable<object[]> EnumeratorTraversalNullData()
 
         [Theory]
         [MemberData(nameof(EqualsData))]
-        public void Equals(ImmutableArray<int> first, ImmutableArray<int> second, bool expected)
+        public void EqualsBaseTest(ImmutableArray<int> first, ImmutableArray<int> second, bool expected)
```

Comment:
Elsewhere you used EqualsTest.  Why EqualsBaseTest here?

Reply:
I thought it had been because of some class that inherited from this and had its own `EqualsTest`, but apparently it's not the case here, I'll simplify to `EqualsTest`.